src/gui/accountsettings.cpp

```diff
@@ -620,14 +620,24 @@ void AccountSettings::slotAccountStateChanged(int state)
     /* Allow to expand the item if the account is connected. */
     ui->_folderList->setItemsExpandable(state == AccountState::Connected);
 
-    /* check if there are expanded root items, if so, close them, if the state is different from being Connected. */
     if (state != AccountState::Connected) {
+        /* check if there are expanded root items, if so, close them */
         int i;
         for (i = 0; i < _model->rowCount(); ++i) {
             if (ui->_folderList->isExpanded(_model->index(i)))
                 ui->_folderList->setExpanded(_model->index(i), false);
         }
+
+        // Pending selective sync changes get dropped
```

Comment:
That's sad :-(

The user did his selection and then the connection drops and the selection is gone?